CIP-0013/CIP-0013.md

```diff
@@ -43,27 +54,64 @@ Example:
 
 ## ABNF Grammar (Proposal)
 
-This is an initial, simplified protocol definition for fast implementation; it only requires an address and an optional amount parameter. As discussed above, these rules are likely to evolve in time in order to support additional features, including unique capabilities of the Cardano blockchain.
+This is an initial, simplified protocol definition for fast implementation; it only requires:
+
+* for a payment URI (authority unspecified), an address and an optional amount parameter;
+* for a stake pool URI (authority = `stake`), a single stake pool reference.
+
+As discussed above, these rules are likely to evolve in time in order to support additional features, including multiple stake pool references (in proportions defining a "portfolio") and other unique capabilities of the Cardano blockchain.
 
 ```
-cardanourn     = "web+cardano:" cardanoaddress [ "?" amountparam ]
+cardanourn = "web+cardano:" (paymentref | stakepoolref)
+
+paymentref = cardanoaddress [ "?" amountparam ]
 cardanoaddress = *(base58 | bech32)
-amountparam    = "amount=" *digit [ "." *digit ]
+amountparam = "amount=" *digit [ "." *digit ]
+
+stakepoolref = "//stake?" stakepool
+stakepool = poolhexid | poolticker
+poolhexid = 56HEXDIG
+poolticker = 3*5UNICODE
 ```
 
+### Payment URI queries
+
 The amount parameter must follow the [same rules](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#transfer-amountsize) described in BIP-21, namely, it must be specified in decimal ADA, without commas and using the period (.) decimal separator.
 
+### Stake pool URI queries
+
+For brevity, essential in many Internet contexts, `poolticker`  must be supported here in addition to the unambiguous `poolhexid`.  A safe interpretation of `poolticker` should be assured by the Daedalus wallet's integration with [SMASH](https://github.com/input-output-hk/smash) metadata.
```

Comment:
> assured by the Daedalus wallet's integration

I would avoid mentioning a specific wallet implementation. This CIP targets all wallets. To be honest, I would remove this entire sentence since SMASH also seems like a detail unrelated to this CIP:

> A safe interpretation of `poolticker` should be assured by the Daedalus wallet's integration with [SMASH](https://github.com/input-output-hk/smash) metadata.

Reply:
All your recommended changes look fine to me... this is my first time editing this way so I will just resolve these sections one by one, editing my own copy, and then push a single commit with my best efforts to accommodate all of them.

Reply:
Have removed that sentence.